src/presentation/ui/component/runtimes/phpRuntimeTabContent.templ

```diff
@@ -0,0 +1,268 @@
+				htmx.ajax(
+					'PUT',
+					'/api/v1/runtime/php/' + vhostHostname + '/',
+					{
+						swap: 'none',
+						values: { version: this.selectedVersion },
+					},
+				);
+				this.closeUpdatePhpVersionModal();
+			},
+		}));
+	});
+}
+
+templ PhpRuntimeTabContent(
+	runtimeOverview presenterDto.RuntimeOverview,
+	existentVhostsHostnames []string,
+) {
+	<!-- PhpRuntimeTabContent -->
+	@PhpRuntimeTabContentLocalState()
+	<div class="bg-os-800 -mt-4 rounded-b-xl rounded-r-xl p-4" x-data="php">
+		<div class="bg-os-500 rounded-b-lg rounded-r-lg px-4 py-2">
+			if runtimeOverview.IsInstalled {
+				<div class="p-4">
+					<div class="float-left w-1/5">
+						@componentForm.SelectInput(
+							"virtualHost", "Virtual Host Hostname",
+							"", runtimeOverview.VirtualHostHostname.String(),
+							"changeSelectedVhostHostname($event.target.value)",
+							existentVhostsHostnames, false,
+						)
+					</div>
+					if runtimeOverview.IsMappingAlreadyCreated {
+						<div class="w-1/7 flex pl-5">
+							@componentForm.SelectInput(
+								"version", "Version", "",
+								runtimeOverview.PhpConfigs.Version.Value.String(),
+								"openUpdatePhpVersionModal($event.target.value)",
+								transformPhpVersionOptionsIntoStringSlice(runtimeOverview.PhpConfigs.Version.Options),
+								false,
+							)
+						</div>
+						<div class="mt-6 flex">
+							@componentStructural.SidebarMenuHeader(
+								[]componentStructural.SidebarMenuHeaderItem{
+									{Label: "Modules", Icon: "ph-puzzle-piece", ComponentValue: "modules"},
+									{Label: "Settings", Icon: "ph-gear", ComponentValue: "settings"},
+								}, "setSelectedConfigType",
+							)
+							<div class="px-10 py-4">
+								@PhpModulesCheckboxInputsSwitchToggles(
+									runtimeOverview.VirtualHostHostname,
+									runtimeOverview.PhpConfigs.Version,
+									runtimeOverview.PhpConfigs.Modules,
+								)
+								@PhpSettingsSelectInputs(
+									runtimeOverview.VirtualHostHostname,
+									runtimeOverview.PhpConfigs.Version,
+									runtimeOverview.PhpConfigs.Settings,
+								)
+							</div>
+						</div>
+					} else {
+						<div class="mb-6 flex w-full"></div>
+						@CreatePhpMappingForm(runtimeOverview.VirtualHostHostname)
+					}
+				</div>
+			} else {
+				@componentStructural.ServiceNotInstalledWarningForm("php")
+			}
+		</div>
+		@UpdatePhpVersionModal(runtimeOverview.VirtualHostHostname)
+	</div>
+}
+
+func transformPhpVersionOptionsIntoStringSlice(
+	versionOptions []valueObject.PhpVersion,
+) []string {
+	versionOptionsStrSlice := []string{}
+	for _, versionOption := range versionOptions {
+		versionOptionsStrSlice = append(versionOptionsStrSlice, versionOption.String())
+	}
+
+	return versionOptionsStrSlice
+}
+
+templ PhpModulesCheckboxInputsSwitchToggles(
+	selectedVhostHostname valueObject.Fqdn,
+	selectedVersion entity.PhpVersion,
+	modules []entity.PhpModule,
+) {
+	<!-- PhpModulesCHeckboxInputsSwitchToggles -->
```

Comment:
I removed the comments. I'm not going to separate this into a specific package for runtime since this piece of code pertains specifically to PHP. I don't know if in future runtimes we'll end up providing modules and settings, and they also carry PHP's typing, so I don't think they're as reusable as the database component, for example. I prefer to keep them within the PhpRuntime scope! :)